### DISCUSSION TITLE
the Bromite SystemWebView guide says it can be installed only if it's already installed

### DISCUSSION THREAD

**Discussion by chaserene** [bromite/bromite#1784] [Q&A]
(right after my [Reddit post](https://www.reddit.com/r/fossdroid/comments/sqffte/the_bromite_systemwebview_guide_says_it_can_be/) I found that csagan5 is no longer active on reddit and that this project has Discussions, so I'm reposting here.)

The [current Bromite SystemWebView installation guide](https://github.com/bromite/bromite/wiki/Installing-SystemWebView/bb382fb6de86035db14330803aca148fe584b832) says:

> You can install Bromite SystemWebView only if the above output contains a packagename="**org.bromite.webview**" with the correct signature block or without a corresponding E: signature block

that seemed weird, since the output in question is that of a system file (`/system/framework/framework-res.apk/res/xml/config_webview_packages.xml`), and my phone so far haven't had anything Bromite-related on it. I checked mine (on the latest LineageOS), and unsurprisingly, it didn't contain `packagename="org.bromite.webview"`. I mean, who would have put it there, the AOSP devs?

the documentation has been saying this only since the last revision made on [February 5, 2022](https://github.com/bromite/bromite/wiki/Installing-SystemWebView/_history). the [revision before](https://github.com/bromite/bromite/wiki/Installing-SystemWebView/24306f815a17857a48a6dfed5ccfdfa5a46673fd) says what makes sense and what I remembered, `packagename="com.android.webview"`.

is this a mistake or am I missing something?

**Comment by csagan5** [comment]
> right after my [Reddit post](https://www.reddit.com/r/fossdroid/comments/sqffte/the_bromite_systemwebview_guide_says_it_can_be/) I found that csagan5 is no longer active on reddit

Says who?

> I mean, who would have put it there, the AOSP devs?

Yes. Or you can build or modify a ROM that way.

> is this a mistake or am I missing something?

See https://www.bromite.org/news/2022-02-05-bromite-webview-package-name-change

The new package name is not yet active.
